lib/Dialect/SV/SVOps.cpp

```diff
@@ -155,7 +155,7 @@ void RegOp::build(OpBuilder &odsBuilder, OperationState &odsState,
 void RegOp::getAsmResultNames(OpAsmSetValueNameFn setNameFn) {
   // If the wire has an optional 'name' attribute, use it.
   auto nameAttr = (*this)->getAttrOfType<StringAttr>("name");
-  if (!nameAttr.getValue().empty())
+  if (nameAttr && !nameAttr.getValue().empty())
```

Comment:
This is an unrelated small fix for a bug I encountered during development.

Reply:
This is (now) required in the sv.reg op, so please drop this:

` let arguments = (ins StrAttr:$name);`

Reply:
You mean from the RegOp ODS? If it's required, why should I delete it?

This was simply to avoid a crash with "--debug" while debugging my lowering pass.

Reply:
I think there are certain functions which should never crash including on invalid IR. It should at least be an assert. It wasn't clear that this was the issue.

Reply:
Sorry, you meant drop the change, not delete the `let arguments = (ins StrAttr:$name);` line from the RegOp ODS, yes?